src/Type/Object/PostObject.php

```diff
@@ -51,99 +72,131 @@ function register_post_object_types( $post_type_object ) {
 		/**
 		 * Register fields custom to the MediaItem Type
 		 */
-		register_graphql_fields( $post_type_object->graphql_single_name, [
-			'caption'      => [
-				'type'        => 'String',
-				'description' => __( 'The caption for the resource', 'wp-graphql' ),
-				'args'        => [
-					'format' => [
-						'type'        => 'PostObjectFieldFormatEnum',
-						'description' => __( 'Format of the field output', 'wp-graphql' ),
+		register_graphql_fields(
+			$post_type_object->graphql_single_name,
+			[
+				'caption'      => [
+					'type'        => 'String',
+					'description' => __( 'The caption for the resource', 'wp-graphql' ),
+					'args'        => [
+						'format' => [
+							'type'        => 'PostObjectFieldFormatEnum',
+							'description' => __( 'Format of the field output', 'wp-graphql' ),
+						],
 					],
+					'resolve'     => function( $source, $args ) {
+						if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+							// @codingStandardsIgnoreLine.
+							return $source->captionRaw;
+						} else {
```

Comment:
this unnecessary `else` block an be omitted

Reply:
fixed